src/views/blocks/tickets/submit-button-modal.php

```diff
@@ -21,20 +24,34 @@
 $button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
 
 /**
- * Filter Modal Content
+ * Allow filtering of the button classes for the tickets block.
+ *
+ * @since TBD
+ *
+ * @param array $button_name The button classes.
+ */
+$button_classes = apply_filters( 'tribe_tickets_ticket_block_submit_classes', [
+	'tribe-common-c-btn',
+	'tribe-common-c-btn--small',
+	'tribe-tickets__buy'
```

Comment:
* :no_entry_sign: Comma required after last value in array declaration
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 4464a4ef22b98e4f740b7b0b0edd9edc</sub>